OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -390,6 +395,13 @@ bool InverseKinematicsTool::run()
                         worst = j;
                     }
                 }
+
+				double rms = sqrt(totalSquaredMarkerError / nm);
+				MarkerErrors.set(0, totalSquaredMarkerError); 
+				MarkerErrors.set(1, rms);
+				MarkerErrors.set(2, sqrt(maxSquaredMarkerError));
+				modelMarkerErrors->append(s.getTime(), 3, &MarkerErrors[0]);
+
                 cout << "Frame " << i << " (t=" << s.getTime() << "):\t";
                 cout << "total squared error = " << totalSquaredMarkerError;
                 cout << ", marker error: RMS=" << sqrt(totalSquaredMarkerError/nm);
```

Comment:
To avoid code duplication, can you replace this calculation with your `rms` variable?